build/appveyor_install.ps1

```diff
@@ -133,7 +133,7 @@ If (!(Test-Path C:\Users\appveyor\.ccm\repository\$env:cassandra_version)) {
 #Download simulacron jar
 $simulacron_path = "$($dep_dir)\simulacron.jar"
 If (!(Test-Path $simulacron_path)) {
-  $simulacron_version = "0.8.2"
+  $simulacron_version = "0.8.11"
```

Comment:
We can jump to 0.9.0, as we will need it soon.

Reply:
done